### DISCUSSION TITLE
Some checkbox of add bookmarks

### DISCUSSION THREAD

**Discussion by LLKoder** [go-shiori/shiori#790] [Q&A]
![2023-11-27_15-36](https://github.com/go-shiori/shiori/assets/50442676/9024e692-f3c0-4567-92dc-fc406b4a1218)

What do mean this checkbox? I didn't find it in documentation.

**Comment by Monirzadeh** [comment]
Archive: Create a replica of the original version of the link.
Ebook: Create an ebook (epub) file from a readable version of URL (without ads, unnecessary parts, etc.) that you can read on ebook readers or mobile phones and tablets.
If you check the last checkbox, anyone can access this bookmark if they have the URL of this bookmark on Shiori (no need to login).
Please close issue if you get answer or even better send a PR and add that to documentation

**Comment by LLKoder** [comment]
I don't understand first case. What I get with Archive and without. To copy bookmarks to archive only?
Last case: where do i get URL of bookmark that I can try this features.

OK. I try to add this in documentation at weekend. I have to understand how these features work before.

**Comment by Monirzadeh** [comment]
Shiori always create a readable version, archive is full version (with all content like [Internet Archive](https://archive.org/) without versioning)
Just create a link with and without first one an see difference.

Bookmark can be public or Private. If a bookmark set public no need to login to access that.
I will convet this to discussion.

**Comment by LLKoder** [comment]
@Monirzadeh I understand archive and ebook. But I don't see changes when bookmark is public. Where do I see public bookmark or where I have to get a link to bookmark to get access to bookmark page if I not login?

**Reply by fmartingr** [reply]
Public is for archives only. The same Archive URL you use should b public without login in to Shiori. (Something like `<URL>/bookmark/X/archive`.
